### PR TITLE
Add test spec

### PR DESCRIPTION
**Pull request by andresilva**
Add js test spec and fix code style.

### CODE REVIEW

**Comment by andresilva** [comment]
I've also updated the ABI json file to only include the ABI itself.

File: abis/GithubHint.json [hunk 1]
```diff
@@ -1,2517 +1,83 @@
...
-    "absolutePath": "/home/sorpaas/repos/github-hint/contracts/GithubHint.sol",
-    "exportedSymbols": {
-      "GithubHint": [
-        95
-      ]
-    },
-    "id": 96,
-    "nodeType": "SourceUnit",
-    "nodes": [
+[
```
Comment (sorpaas) [inline abis/GithubHint.json:1]: Ooops! Looks like I generated ABI files in the wrong way.

**Review by sorpaas — APPROVED** [review]

File: abis/GithubHint.json (hunk 1 above)
Comment (andresilva) [reply abis/GithubHint.json:1]: `cat build/contracts/GithubHint.json | jq .abi > abis/GithubHint.json`

**Review by kirushik — APPROVED** [review]
Ĺ͘͝Ǵ̨T̵̢͞͝M͢!̷

**Pull request merged**